app/theme/client/main.css

```diff
@@ -58,3 +58,344 @@
 
 /* RTL */
 @import 'imports/general/rtl.css';
+
+/* Visitor History CSS */
+.inner-addon{
+	position: relative;
+	left: 24px;
+	width: 345px;
+	height: 40px;
+	background: #ffffff;
+	border: 2px solid #CBCED1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#CBCED1" to be "#cbced1" (color-hex-case)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

Reply:
done please check